app/assets/stylesheets/views/_checkView.scss

```diff
@@ -0,0 +1,77 @@
+@import 'utilities/variables';
```

Comment:
this import is not necessary if declare inside of the `app.scss` file

Reply:
done